frame/staking/src/mock.rs

```diff
@@ -263,6 +267,8 @@ impl Config for Test {
 	type ElectionProvider = onchain::OnChainSequentialPhragmen<Self>;
 	type GenesisElectionProvider = Self::ElectionProvider;
 	type WeightInfo = ();
+	type VoterBagThresholds = VoterBagThresholds;
+	type BagIdx = u8;
```

Comment:
Need to generate the threshold list, get rid of `BagIdx`, before merging. In short, make the tests pass.

Reply:
They pass now.